docs/tutorials/quick_start_rake_tasks.md

```diff
@@ -0,0 +1,205 @@
+#Beaker Quick Start Tasks
+
+We have developed some rake tasks to help new Beaker users get up and running quickly with writing and running tests.
+
+## Summary
+
+### Gen tasks
+
+These tasks are standalone and can be run independently from each other to generate the desired files.
+
+beaker_quickstart:gen_hosts  (generates default host config)
+beaker_quickstart:gen_pre_suite  (generates default pre-suite)
+beaker_quickstart:gen_smoke_test  (generates default smoke test)

```

Comment:
@pinkypie when I view these rendered, they don't show up in a list. Was this intended?